services/emailer/poetry.lock

```diff
@@ -1531,7 +1531,7 @@ zipp = "^3.8.1"
 type = "git"
 url = "https://github.com/bcgov/namex.git"
 reference = "HEAD"
-resolved_reference = "63b06cfb82228002e89e7f43343c117cde4a0ae5"
+resolved_reference = "0a7ac33807f801b73f3d08171213e1fccb343a38"
```

Comment:
FYI, this came from here:

<img width="872" alt="Screenshot 2025-01-10 at 4 20 32 PM" src="https://github.com/user-attachments/assets/c4c9b2f9-3318-4b96-bf06-615fcb2c39b8" />